cypress/plugins/synapsedocker/index.ts

```diff
@@ -101,12 +101,13 @@ async function synapseStart(template: string): Promise<SynapseInstance> {
 
     const synapseId = await dockerRun({
         image: "matrixdotorg/synapse:develop",
-        containerName: `react-sdk-cypress-synapse-${crypto.randomBytes(4).toString("hex")}`,
+        containerName: `react-sdk-cypress-synapse`,
         params: [
             "--rm",
             "-v", `${synCfg.configDir}:/data`,
             "-p", `${synCfg.port}:8008/tcp`,
         ],
+        cmd: "run",
```

Comment:
Does that work? `dockerRun` has `run` as the first arg. Passing it in `cmd` appends it to the arg list.
https://github.com/matrix-org/matrix-react-sdk/pull/8328/files#diff-38e25250efba2fc46fdce7da7094ce9f2835e99c185ce69460b57ac7cb5b9b2cR51

Reply:
This is `run` for Synapse: https://hub.docker.com/r/matrixdotorg/synapse/ https://github.com/matrix-org/synapse/blob/develop/docker/start.py#L220 - `run` is implicit if unspecified

Reply:
Ok, I see :+1: